test/basic_tests.py

```diff
@@ -13,13 +11,15 @@
 import sys
 
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), "../opt/readies"))
-import paella
```

Comment:
Better to keep it. It is useful for debugging purposes (enables easily debugger invocation and insertion of breakpoints in code).

Reply:
ok